backend/app/api/endpoints/knowledge.py

```
@@ -767,3 +810,202 @@ def get_qa_history(
+    # Run in background, return immediately
+    background_tasks.add_task(
+        _run_document_summary_refresh, doc_id, current_user.id, db
+    )
+
+    return SummaryRefreshResponse(
+        message="Summary refresh started",
+        status="generating",
+    )
+
+
+async def _run_kb_summary_refresh(kb_id: int, user_id: int, db: Session):
```

Ruff (0.14.10)

Unused function argument: `db`

(ARG001)